src/Exceptions/HttpMethodNotAllowed.php

```diff
@@ -8,4 +8,3 @@ class HttpMethodNotAllowed extends HttpException {
     public const DESCRIPTION = "Indicates that the method specified in the request-line is known by the origin server but not supported by the target resource.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/405";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces